test/e2e/common_test.go

```diff
@@ -203,6 +223,13 @@ var _ = SynchronizedAfterSuite(func() {
 	timingsFile = nil
 },
 	func() {
+		// perform a netns leak check after all tests run
+		newNetnsFiles, err := os.ReadDir(getNetnsDir())
+		if !errors.Is(err, fs.ErrNotExist) {
+			Expect(err).ToNot(HaveOccurred())
+		}
+		Expect(newNetnsFiles).To(ConsistOf(netnsFiles), "Netns files were leaked")
```

Comment:
This will trigger (confusingly) if a netns file vanishes between t_start and t_end. I don't think this is worth the time I spent testing it, but I did anyway.

Reply:
you used diff in the system tests which also fails if you remove a file so I didn't consider this as a problem.
I think it is fine like this but we can fix it is seems to cause issues in the wild.